build.sbt

```diff
@@ -1,7 +1,8 @@
 import sbt.Keys._
 import sbt._
 
-name := "codacy-metrics-gocyclo"
```

Comment:
This make no sense. You just need the name inside the project. Why have it twice.